src/routes/+page.svelte

```diff
@@ -307,6 +334,29 @@
 	</div>
 </Section>
 
+<!-- Our Values -->
+<Section id="values" class="relative py-20">
+	<svelte:fragment slot="title">
+		<!-- eslint-disable-next-line svelte/no-at-html-tags -->
+		{@html c(i("home.values.title"))}
+	</svelte:fragment>
+	<div class="flex items-center justify-between">
+		<div class="grid gap-x-16 gap-y-12 pb-8 pt-4 sm:grid-cols-2">
+			{#each values as value}
+				<div class="grid grid-flow-col gird gap-x-4 items-start justify-start">
```

Comment:
`gird` :D did you run Prettier locally?

Reply:
Also, is `grid-flow-col` really useful here? I'm not sure but I can be wrong (It's a 2x2 grid, flow doesn't matter I guess)

Reply:
this is not for 2x2, it's necessary to align the icon and text. I prefer to use a grid because it takes the height and width.

Reply:
Hum,i had run prettier but it doesn't seem to work 🤔

Reply:
I found a solution. I have to add in settings of vs code svelte editor format -> svelte vs code 
And now on save prettier runs